.github/workflows/vast.yaml

```diff
@@ -67,10 +67,10 @@ jobs:
       cmake-version-args: ${{ steps.configure.outputs.cmake-version-args }}
       docker-build-args: ${{ steps.configure.outputs.docker-build-args }}
```

Comment:
These need to be updated as well for the CE versions because they show the wrong version string currently, unless the goal is to also fix this in the follow-up by removing this logic entirely.

```
❯ docker run --platform=linux/amd64 ghcr.io/tenzir/vast-ce:topic-flake-version version
<jemalloc>: MADV_DONTNEED does not work (memset will be used instead)
<jemalloc>: (This is the expected behaviour if you are running under QEMU)
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-netflow.so
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-sigma.so
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-pcap.so
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-web.so
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-matcher.so
[13:41:18.133] loaded plugin: /opt/tenzir/vast/lib/vast/plugins/libvast-plugin-parquet.so
{
  "VAST": "v2.4.0-1081-g25588c556c",
  "Build Configuration": {
    "Type": "Release",
    "Tree Hash": "9bbefa8033590ed7dc15be3806f8caf0",
    "Assertions": false,
    "Address Sanitizer": false,
    "Undefined Behavior Sanitizer": false
  },
  "CAF": "0.18.7",
  "Apache Arrow": "11.0.0",
  "jemalloc": null,
  "plugins": {
    "matcher": "v3.0.0-g",
    "netflow": "v1.0.0-g",
    "parquet": "v1.0.0-g601159c6b4",
    "pcap": "v1.1.0-ge4e37142e6",
    "sigma": "v1.1.0-ge4e37142e6",
    "web": "v1.0.0-g25588c556c"
  }
}
```

Reply:
I had a quick look. Ugh, I'd rather not add the submodule checkout to the configure script only to remove it next week again.

Reply:
Yeah, then just do it as part of the plugin version removal.